omod/src/main/resources/config.xml

```diff
@@ -12,4 +12,8 @@
 
 	<require_version>${openmrsVersion}</require_version>
 
+	<aware_of_modules>
+		<aware_of_module>org.openmrs.event</aware_of_module>
+	</aware_of_modules>
+
```

Comment:
We can remove this if we decide we don't want to support the example Listener that I included as a POC